single_stage_detector/README.md

```diff
@@ -0,0 +1,293 @@
+           detections.csv
+           openimages-mlperf.json
+```
+
+Read more about the mlperf subset [here](#the-mlperf-subset).
+
+
+## Download the pretrained backbone
+The benchmark uses a ResNeXt50_32x4d backbone pretrained on ImageNet. The
+weights are downloaded from PyTorch hub.
+
+By default, the code will automatically download the weights to
+`$TORCH_HOME/hub` (default is `~/.cache/torch/hub`) and save them for later use.
+
+Alternatively, you can manually download the weights with:
+```bash
+bash ./single_stage_detector/scripts/download_backbone.sh
+```
+
+Then use the downloaded file with `--pretrained <PATH TO WEIGHTS>` .
+
+If you wish to use the backbone in frameworks other than PyTorch, the repository
+provides scripts to convert the weights to ONNX format and a pickled dictionary
+of numpy arrays:
+```
+./single_stage_detector/scripts/pth_to_onnx.py --help
+./single_stage_detector/scripts/pth_to_pickle.py --help
+```
+
+## Training on NVIDIA DGX-A100 (single node)
+Launch configuration and system-specific hyperparameters for the NVIDIA DGX-A100
+single node reference are in the `config_DGXA100_001x08x032.sh` script.
+
+Steps required to launch single node training on NVIDIA DGX-A100:
+
+```
+cd ./single_stage_detector/ssd
+source config_DGXA100_001x08x032.sh
+DATADIR=<path/to/data/dir> LOGDIR=<path/to/output/dir> ./run.sub
+```
+
+## Training on NVIDIA DGX-A100 (multi node)
+Launch configuration and system-specific hyperparameters for the NVIDIA DGX-A100
+multi node reference are in the `config_DGXA100_008x08x008.sh` and
+`config_DGXA100_032x08x032.sh` scripts.
+
+Steps required to launch multi node training on NVIDIA DGX-A100:
+
+```
+cd ./single_stage_detector/ssd
+source config_DGXA100_008x08x008.sh # or config_DGXA100_032x08x032.sh
+DATADIR=<path/to/data/dir> LOGDIR=<path/to/output/dir> sbatch -N $DGXNNODES -t $WALLTIME run.sub
+```
+
+## Hyperparameter settings
+Hyperparameters are recorded in the `config_*.sh` files for each configuration
+and in `run_and_time.sh`.
+
+# Dataset/Environment
+## Publication/Attribution
+[Google Open Images Dataset V6](https://storage.googleapis.com/openimages/web/index.html)
+
+## The MLPerf Subset
+The MLPerf subset includes only 264 classes of the total 601 available in the
+full dataset:
+
+| Dataset           | # classes | # train images | # validation images | Size  |
+|-------------------|-----------|----------------|---------------------|-------|
+| OpenImages Full   | 601       | 1,743,042      | 41,620              | 534GB |
+| OpenImages MLperf | 264       | 1,170,301      | 24,781              | 352GB |
+
+The list of used classes can be viewed
+[here](https://github.com/mlcommons/training/blob/master/single_stage_detector/scripts/download_openimages_mlperf.sh).
+
+# Model
+This network takes an input 800x800 image from [OpenImages-v6](https://storage.googleapis.com/openimages/web/index.html)
+and 265 categories, and computes a set of bounding boxes and categories.
```

Comment:
is this a typo? should it be 264 categories?

Reply:
corrected to 264